save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -467,6 +468,11 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
         }
     }
 
+    private fun getUrlWithFiltersParams(filterValue: TestExecutionFilters): String {
+        val hrefBeforeValues = window.location.href.substringBefore("?")
+        return "$hrefBeforeValues${filterValue.toQueryParams()}"
+    }
```

Comment:
```suggestion
private fun getUrlWithFiltersParams(filterValue: TestExecutionFilters) = "${window.location.href.substringBefore("?")}${filterValue.toQueryParams()}"
```

maybe it might be nicer?

Reply:
Done